lib/cdo/honeybadger.rb

```diff
@@ -0,0 +1,61 @@
+require 'honeybadger'
+require_relative 'env'
+
+# Honeybadger extensions for command error logging
+module Honeybadger
+  class << self
+
+  # notify_command_error - log an error from an executed command to honeybadger.
```

Comment:
Oh yeah, it should be indented. That was an oversight.

Anyway, I'll switch to self.etc